core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -1135,4 +1147,69 @@ public synchronized void updateGroup() {
       mRaftGroup = newGroup;
     }
   }
+
+  private RaftProtos.RoleInfoProto getRaftRoleInfo() {
+    GroupInfoReply groupInfo = null;
+    try {
+      groupInfo = getGroupInfo();
+    } catch (IOException e) {
+      LOG.error("Error while get group info", e);
+    }
+    if (groupInfo == null || groupInfo.getException() != null) {
+      return null;
+    }
+    return groupInfo.getRoleInfoProto();
+  }
+
+  /**
+   * Get the role name.
```

Comment:
nit: please update the comments to be more informative, like giving examples what the values you are getting

Reply:
Added a link